pycode/memilio-epidata/memilio/epidata/customPlot.py

```diff
@@ -37,36 +45,47 @@ def plotList(x, y, legend, title='', xlabel='', ylabel='', fig_name=''):
     @param title title of the plot.
     @param xlabel labels for x values.
     @param ylabel labels for y values.
+    @param linewidth [Default: 1] Width of plotted curves.
+    @param xticks_idx [Default: 'default'] List of xticks indices to be used. 
+        If 'default' is used, then every 30th x-value is printed.
+    @param dpi Dots per inch of figure.
+    @param outercolor Outer color of figure.
+    @oaram innercolor Inner color of figure.
     @param fig_name name of the figure to save.
-    @return dataframe with imputed dates (and moving average if requested)
     """
-    if len(y) > 3:
-        fig, ax = plt.subplots(figsize=(10, 6))
-    else:
-        fig, ax = plt.subplots(figsize=(8, 6))
+    fig, ax = plt.subplots(figsize=fig_size, facecolor=outercolor)
 
     for i in range(len(y)):
-        ax.plot(x, y[i], label=legend[i])
+        ax.plot(x, y[i], linewidth=linewidth, label=legend[i])
     if title != '':
         ax.set_title(title, fontsize=18)
-    tick_range = (np.arange(int(len(x) / 30) + 1)
-                  * 30)  # only plot every 30th day
-    tick_range[-1] -= 1  # adapt for last element
+
+    ax.set_facecolor(innercolor)
+
     if isinstance(x, pd.Series):
         x = x.values
-    ax.set_xticks(x[tick_range])
-    ax.set_xticklabels(x[tick_range], rotation=90, fontsize=8)
+    if xticks_idx == 'default':
+        tick_range = (np.arange(int(len(x) / 30) + 1)
+                      * 30)  # only plot every 30th day
+        tick_range[-1] -= 1  # adapt for last element
+        ax.set_xticks(x[tick_range])
```

Comment:
Isn't tick_range a list and no indice? Should it be ax.set_xticks([x[tick_range[i]] for i in tick_range])?